buddy_init/init_actions.py

```diff
@@ -0,0 +1,104 @@
+import logging
+import os
+import shutil
+import subprocess
+
+import fabric
+import paramiko
+from invoke import UnexpectedExit
+
+SSH_TIMEOUT = 10
+logger = logging.getLogger(__name__)
+
+
+def setup_ssh():
+    config_file = os.path.expanduser('~/.ssh/config')
+    if not os.path.exists(config_file):
+        logger.info("SSH config file not found. Creating a new one.")
+        with open(config_file, "w"):
+            pass
+    else:
+        shutil.copy(config_file, config_file + ".bak")
+        logger.info(f"Found config file. Saved as {config_file}.bak.")
+
+    with open(config_file) as fin:
+        current_config = fin.read()
+
+    if 'Host mila' not in current_config:
+        print("""One time setup to connect with the Mila servers""")
+        wrong_username = True
+        while wrong_username:
+            mila_username = input("What is your mila username?")
+            wrong_username = enforce_answer_list(f"Your cluster username is {mila_username}", ["y", "n"]) == "n"
+
+        with open(config_file, 'a') as fout:
+            fout.write(
+                "Host mila\n"
```

Comment:
an initial \n could be needded (not sure)

Reply:
+1